stdlib/Makefile

```diff
@@ -2,18 +2,23 @@ CC=clang
 BIT_INT_FLAGS=-Xclang -fexperimental-max-bitint-width=512
 CFLAGS=$(TARGET_FLAGS) -emit-llvm -O3 -ffreestanding -fno-builtin -Wall -Wno-unused-function $(BIT_INT_FLAGS)
 
-bpf/%.bc wasm/%.bc: %.c
+bpf/%.bc wasm/%.bc: %.c | outputs_dirs
```

Comment:
Does this work on pattern recipe? I would expect this
```
$(SOLANA) $(WASM): | outputs_dirs
```
If it works, then fine of course :)